cli/cmd/environment.go

```diff
@@ -31,6 +34,22 @@ Or just save it to a file
 		username := os.Getenv("FASIT_USERNAME")
 		password := os.Getenv("FASIT_PASSWORD")
 
+		// Fall back to the ident of the logged in user
+		if username == "" {
+			username = os.Getenv("USER")
```

Comment:
Would it be more natural to also ask for the username, and set it to `$USER` as default, so you could just press return?

```
Username (default $USER): 
```

Reply:
I what circumstances is `$USER` wrong? I would prefer to "silently" default to `$USER` as this removes the need to press Return 99 % of the times.

Reply:
I misread the code. I agree that this makes sense